packages/opal-server/requires.txt

```diff
@@ -5,3 +5,6 @@ pyjwt[crypto]>=2.1.0,<3
 websockets>=10.3,<11
 ddtrace>=1.1.4,<2
 slowapi>=0.1.5,<2
+# slowapi is stuck on and old `redis`, so fix that and switch from aioredis to redis
+aioredis
+celery
```

Comment:
please limit the versions of the libraries you add to opal.

assuming the current version of aioredis is 2.0.1, you want to do this:
```
aioredis>=2.0.1<3
```
to make sure once a backwards incompatible change is introduced - it won't affect this version of OPAL.